tiledb/sm/query/query.cc

```diff
@@ -1063,6 +1090,12 @@ Status Query::set_subarray_unsafe(const NDRange& subarray) {
   return Status::Ok();
 }
 
+const Subarray* Query::subarray() {
```

Comment:
This might have to go away completely in your rebase.

Reply:
'dev' merged